gum/backend-darwin/gumobjcapiresolver.c

```diff
@@ -171,38 +171,58 @@ gum_objc_api_resolver_enumerate_matches (GumApiResolver * resolver,
   GumObjcApiResolver * self = GUM_OBJC_API_RESOLVER (resolver);
   GMatchInfo * query_info;
   gchar method_type;
-  GPatternSpec * class_spec, * method_spec;
+  GPatternSpec  * modifiers_spec, * class_spec, * method_spec;
+  gboolean ignore_case = FALSE;
   GHashTableIter iter;
   gboolean carry_on;
   GHashTable * visited_classes;
   GumObjcClassMetadata * klass;
-
+  
   g_regex_match (self->query_pattern, query, 0, &query_info);
   if (!g_match_info_matches (query_info))
     goto invalid_query;
 
   method_type = gum_method_type_from_match_info (query_info, 1);
-  class_spec = gum_pattern_spec_from_match_info (query_info, 2);
-  method_spec = gum_pattern_spec_from_match_info (query_info, 3);
+  
+  modifiers_spec = gum_pattern_spec_from_match_info (query_info, 4, ignore_case);
+  if (modifiers_spec)
```

Comment:
```suggestion
  if (modifiers_spec != NULL)
```

As per the style guide.